pom.xml

```
@@ -83,6 +83,7 @@
         <module>jsonschema-generator</module>
         <module>jsonschema-module-jackson</module>
         <module>jsonschema-module-javax-validation</module>
+        <module>jsonschema-module-jakarta-validation</module>
```

Let's stick to the consistent alphabetic order. Although it's irrelevant. 😄
```suggestion
        <module>jsonschema-module-jakarta-validation</module>
        <module>jsonschema-module-javax-validation</module>
```